app/views/directives/pods-table.html

```diff
@@ -39,11 +39,11 @@
       <td data-title="Age"><span am-time-ago="pod.metadata.creationTimestamp" am-without-suffix="true"></span></td>
       <td ng-if="activePods" data-title="Receiving Traffic">
         <span ng-if="activePods[pod.metadata.name]">
-          <status-icon status="'Succeeded'" disable-animation></status-icon>
+          <status-icon fixed-width="true" status="'Succeeded'" disable-animation></status-icon>
         </span>
         <span ng-if="!activePods[pod.metadata.name]">
            <span data-toggle="popover" data-trigger="hover" data-content="{{podFailureReasons[pod.status.phase] || 'This pod has no endpoints and is not accepting traffic.'}}" style="cursor: help; padding-left: 5px;">
```

Comment:
i think your problem is the padding-left that is on this span, thats what is offsetting this icon compared to the other